src/service/resource.js

```diff
@@ -672,6 +700,9 @@ export class Resource {
    * @return {!../layout-rect.LayoutRectDef}
    */
   getLayoutBox() {
+    if (this.loaderRectV2_) {
```

Comment:
Does v2 not need to manually move `fixed` elements? What about `intersect-resources`?